src/population_loader.rs

```diff
@@ -0,0 +1,109 @@
+use crate::parameters_loader::Parameters;
+use ixa::{
+    context::Context,
+    define_person_property, define_person_property_with_default,
+    error::IxaError,
+    global_properties::ContextGlobalPropertiesExt,
+    people::{ContextPeopleExt, PersonId},
+};
+
+use serde::Deserialize;
+use std::path::PathBuf;
+
+#[derive(Deserialize, Debug)]
+#[allow(non_snake_case)]
+pub struct PeopleRecord<'a> {
+    age: u8,
+    homeId: &'a [u8],
+}
+
+define_person_property!(Age, u8);
+define_person_property!(HomeId, usize);
+define_person_property_with_default!(Alive, bool, true);
+
+define_person_property!(CensusTract, usize);
+
+fn create_person_from_record(
+    context: &mut Context,
+    person_record: &PeopleRecord,
+) -> Result<PersonId, IxaError> {
+    let tract: usize = String::from_utf8(person_record.homeId[..11].to_owned())
+        .expect("Home id should have 11 digits for tract + home id")
```

Comment:
Home ID should have 15 digits, the census tract itself is 11 digits and home ID subcomponent is four.

Reply:
I would let this to be handled in the population skeleton branch